src/core/stream_recv.c

```diff
@@ -793,18 +796,19 @@ QuicStreamOnBytesDelivered(
                 // low.
                 //
 
-                QuicTraceLogStreamVerbose(
-                    IncreaseRxBuffer,
-                    Stream,
-                    "Increasing max RX buffer size to %u (MinRtt=%llu; TimeNow=%llu; LastUpdate=%llu)",
-                    Stream->RecvBuffer.VirtualBufferLength * 2,
-                    Stream->Connection->Paths[0].MinRtt,
-                    TimeNow,
-                    Stream->RecvWindowLastUpdate);
-
-                QuicRecvBufferIncreaseVirtualBufferLength(
-                    &Stream->RecvBuffer,
-                    Stream->RecvBuffer.VirtualBufferLength * 2);
+                if (QuicRecvBufferTryIncreaseVirtualBufferLength(
```

Comment:
Since you're skipping above, do we need the `Try` part anymore, or the failure case handling?

Reply:
It can probably be replaced with an assertion now - especially if we accept the stream knows about the buffer mode.

Reply:
Thanks. I'd prefer that.